pkgs/tools/misc/citron/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, rustPlatform
+, fetchCrate
+, dbus
+, installShellFiles
+, pkg-config
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "citron";
+  version = "0.15.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-6wJ4UfiwpV9zFuBR8SYj6eBiRqQitFs7wRe5R51Z3SA=";
+  };
+
+  cargoHash = "sha256-xTmhgE4iHydhZBMrHWqQUcS9KDlZAzW2CmPGpJr40Fw=";
+
+  buildInputs = [ dbus ];
+
+  nativeBuildInputs = [ installShellFiles pkg-config ];
+
+  postInstall = ''
+    installManPage doc/citron.1
+  '';
+
+  meta = with lib; {
+    homepage = "https://git.sr.ht/~grtcdr/citron";
+    description = "system data via on-demand notifications";
```

Comment:
```suggestion
    description = "System data via on-demand notifications";
```

Reply:
Docs are a bit ambiguous on the first letter capitalisation in the description string:
https://nixos.org/manual/nixpkgs/stable/#var-meta-description

> description
> A short (one-line) description of the package. This is shown by nix-env -q --description and also on the Nixpkgs release pages.
>
>Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat > the name of package — just describe what it does.
>
> Wrong: "libpng is a library that allows you to decode PNG images."
> 
> Right: "A library for decoding PNG images"